packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -120,17 +106,14 @@ export default class QuanticSearchBox extends LightningElement {
 
   disconnectedCallback() {
     this.unsubscribe?.();
-    this.removeEventListener(
-      'suggestionlistrender',
-      this.handleSuggestionListEvent
-    );
   }
 
   updateState() {
     if (this.state?.value !== this.searchBox.state.value) {
-      this.input.value = this.searchBox.state.value;
+      // @ts-ignore
+      this.quanticSearchBoxInput.inputValue = this.searchBox.state.value;
```

Comment:
This line shouldn't be necessary as long as you are passing the `value` as a property to the QuanticSearchBoxInput component like so: 
```
  <c-quantic-search-box-input
    textarea={textarea}
    input={input}
    without-submit-button={withoutSubmitButton}
    placeholder={placeholder}
    search-box-engine={searchBox}
    value={value}
    suggestions={suggestionsArray}
  ></c-quantic-search-box-input>
```